test/jit/test_python_bindings.py

```diff
@@ -0,0 +1,38 @@
+import torch
+from torch.testing._internal.jit_utils import JitTestCase
+
+if __name__ == "__main__":
+    raise RuntimeError(
+        "This test file is not meant to be run directly, use:\n\n"
+        "\tpython test/test_jit.py TestPythonBindings\n\n"
+        "instead."
+    )
+
+
+class TestPythonBindings(JitTestCase):
+    def test_get_python_cu(self):
+        # tests get_python_cu
+        @torch.jit.script
+        def test_get_python_cu_fn(x: torch.Tensor):
+            return 2 * x
+
+        cu = torch._C._jit_get_python_cu()
+        self.assertTrue(
+            "test_get_python_cu_fn" in (str(fn.name) for fn in cu.get_functions())
+        )
+
+    def test_cu_create_function(self):
+        @torch.jit.script
+        def fn(x: torch.Tensor):
+            return 2 * x
+
+        cu = torch._C.CompilationUnit()
+        cu.create_function("test_fn", fn.graph)
```

Comment:
I think more discussion is required on an API like this. I would prefer to include in this pull request only the changes necessary to merge the two implementations.

Reply:
Yeah, so this was actually my starting point in this project that I wanted to create a function from a graph.
Then I saw the split and tried to fix it.
I'm looking into doing building graphs from Python and so making a function from the transformed graph makes that exercise so much more useful.

Reply:
Makes sense, I think this is my second comment about this and the discussion on the other comment was convincing enough for me.